### DISCUSSION TITLE
Chrome Latest Version Problem

### DISCUSSION THREAD

**Discussion by mxignister** [aclap-dev/video-downloadhelper#590] [Help and Q&A]
### Your question

I'm a registered user for Chrome. I noticed the extension was automatically updated. After using it, I encountered multiple download issues that I never encountered in the previous version. Is there a way to revert to the old version?

### Addon & CoApp version and details

Version 9.0.1.13 (stable), target google, locale en-US.

**Comment by paulrouget** [comment]
Can you describe your download issues?

1. make sure to use the latest Chrome version
2. In the `settings > more settings`, there's a "copy to clipboard" button on the top right, can you paste it here?
3. at the bottom of the setting, there's a Use Legacy UI if you want to revert to the old UI
